CanHazFunny/CanHazFunny/Jester.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace CanHazFunny;
+
+    public class Jester 
+    {   
+        
+        private IJokeService? jokeService { get; set; } // set in constructor
```

Comment:
No need to make these nullable since you're checking for null values in the constructor.
```suggestion
        private IJokeService jokeService { get; set; } // set in constructor
```